src/adaptors/hmx/addresses.json

```
@@ -1,12 +1,21 @@
 {
-  "WMATIC": "0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270",
-  "WETH": "0x7ceb23fd6bc0add59e62ac25578270cff1b9f619",
-  "WBTC": "0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6",
-  "DAI": "0x8f3cf7ad23cd3cadbd9735aff958023239c6a063",
-  "USDC": "0x2791bca1f2de4661ed88a30c99a7a9449aa84174",
-  "USDT": "0xc2132d05d31c914a87c6611c10748aeb04b58e8f",
-  "PLP": "0x6A69561199E01E5E76eA53A59aD804d919e30BA8",
-  "POOL_DIAMOND_CONTRACT":"0xE7D96684A56e60ffBAAe0fC0683879da48daB383",
+  "ARB": "0x912CE59144191C1204E64559FE8253a0e49E6548",
+  "DAI": "0xda10009cbd5d07dd0cecc66161fc93d7c9000da1",
+  "GLP": "0x5402B5F40310bDED796c7D0F3FF6683f5C0cFfdf",
+  "HLP": "0x4307fbDCD9Ec7AEA5a1c2958deCaa6f316952bAb",
+  "HMX": "0x83d6c8C06ac276465e4C92E7aC8C23740F435140",
+  "ESHMX": "0x8a011EF14a92AA37cE82A4c95004a70730b6AC38",
+  "WBTC": "0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f",
+  "WETH": "0x82af49447d8a07e3bd95bd0d56f35241523fbab1",
+  "USDC": "0xff970a61a04b1ca14834a43f5de4533ebddb5cc8",
+  "USDT": "0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9",
+  "DRAGON_POINT": "0x26e7DB328167Aef8D8231135F444d0674Cb466fc",
   "PROTOCOL_REVENUE_REWARDER": "0x1fBdB30f4588589575AE2DBbA028CA6f6e8c84dd",
-  "PLP_STAKING":"0x903E50c494A97A133204F5ff207E73e481441064"
-}
+  "FEEDABLE_REWARDER_ESHMX": "0x6D2c18B559C5343CB0703bB55AADB5f22152cC32",
+  "FEEDABLE_REWARDER": "0x665099B3e59367f02E5f9e039C3450E31c338788",
```

```suggestion
  "FEEDABLE_REWARDER": "0x665099B3e59367f02E5f9e039C3450E31c338788",
```
More specific name. There gonna be more rewarders.